index.js

```diff
@@ -9,8 +9,26 @@ class Tree {
   constructor(node) {
     this.root = node;
   }
+  findNode(data) {
+    let currentNode = this.root;
+    let queue = [this.root];
+    let item = null
+    
+    while (queue.length) {
+      currentNode = queue.shift();
+      if (currentNode.data === data) {
+        item = currentNode;
+      }
+
+      if (currentNode.children.length > 0) {
+        for (let child of currentNode.children) {
+          queue.push(child);
+        } 
+      }
+    }
+    return item
+  }
```

Comment:
If you get some spare time Jede && Mason see if you can implement a binary search.  It's very similar to what we have here.